src/Machine.js

```diff
@@ -38,6 +42,13 @@ const Errors = {
 // maximum nesting depth
 const MAX_EXECUTION_DEPTH = 256;
 
+// cache params
+const DEFAULT_EXCLUDE_FILE_NAME = 'builder-cache.exclude';
+const CACHED_READERS = [GithubReader, HttpReader];
+
+const GITHUB_DIR = '/github';
```

Comment:
Whys:

1. Why is these constants for? I only see it's used once.
2. Why two of them?
3. Why to separate cache for github and http resources at all? Even if so, what's about https?
4. Why does it have a platform specific path segments separator ('/') hardcoded here?

I feel like there's something completely wrong here.